src/org/pentaho/di/trans/steps/cassandrasstableoutput/writer/CQL2SSTableWriter.java

```diff
@@ -0,0 +1,177 @@
+/*******************************************************************************
+ *
+ * Pentaho Big Data
+ *
+ * Copyright (C) 2002-2013 by Pentaho : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.pentaho.di.trans.steps.cassandrasstableoutput.writer;
+
+/*
+ * Adapted from DataStax DataImportExample
+ * http://www.datastax.com/wp-content/uploads/2011/08/DataImportExample.java
+ *
+ * Original Disclaimer:
+ * This file is an example on how to use the Cassandra SSTableSimpleUnsortedWriter class to create
+ * sstables from a csv input file.
+ * While this has been tested to work, this program is provided "as is" with no guarantee. Moreover,
+ * it's primary aim is toward simplicity rather than completness. In partical, don't use this as an
+ * example to parse csv files at home.
+ *
+ */
+
+import org.apache.cassandra.db.marshal.AsciiType;
+import org.apache.cassandra.dht.IPartitioner;
+import org.apache.cassandra.dht.Murmur3Partitioner;
+import org.apache.cassandra.io.sstable.SSTableSimpleUnsortedWriter;
+import org.pentaho.di.core.Const;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.i18n.BaseMessages;
+
+import java.io.File;
+import java.nio.ByteBuffer;
+import java.util.Date;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static org.apache.cassandra.utils.ByteBufferUtil.bytes;
+
+/**
+ * Outputs Cassandra SSTables (sorted-string tables) to a directory.
+ *
+ * Adapted from DataStax DataImportExample http://www.datastax.com/wp-content/uploads/2011/08/DataImportExample.java
+ *
+ * @author Rob Turner (robert{[at]}robertturner{[dot]}com{[dot]}au)
+ */
+class CQL2SSTableWriter extends AbstractSSTableWriter {
```

Comment:
This class should have its own unit test class to validate just its functionality.

Also, this looks like it was copied from someone outside of Pentaho. Do we have the rights to do this?
@mbatchelor